Collection.js

```diff
@@ -76,7 +76,7 @@ export default class Collection {
       return false;
     }
     const edvDoc = await this._getEdvDocument({id: existing.id});
-    return edvDoc.delete();
+    return edvDoc.delete({doc: edvDoc});
```

Comment:
I don't think this is right. I think the `doc` it is expecting is not an `EdvDocument` instance, rather, the JSON document that has the sequence number on it, etc. If this is passing tests, I suspect our tests are faulty. Shouldn't it be this?

```suggestion
    return edvDoc.delete({doc: existing});
```

Reply:
@dlongley I misunderstood but now it makes sense. There are no tests currently in the repo, so, I'm using wallet/issuer to test this. you're right the change I made doesn't work. I've updated it to changes suggested. Thanks.